core/node/libp2p/rcmgr_defaults.go

```diff
@@ -39,9 +39,10 @@ func adjustedDefaultLimits(cfg config.SwarmConfig) rcmgr.DefaultLimitConfig {
 		maxconns := cfg.ConnMgr.HighWater
 		if 2*maxconns > defaultLimits.SystemBaseLimit.ConnsInbound {
 			// adjust conns to 2x to allow for two conns per peer (TCP+QUIC)
-			defaultLimits.SystemBaseLimit.ConnsInbound = logScale(2 * maxconns)
-			defaultLimits.SystemBaseLimit.ConnsOutbound = logScale(2 * maxconns)
-			defaultLimits.SystemBaseLimit.Conns = logScale(4 * maxconns)
+			// and 2x again to allow for burst of conns during accelerated DHT client rt refresh
```

Comment:
This PR should include https://github.com/libp2p/go-libp2p-kad-dht/pull/772/. As a way of detecting errors.

For example, by bumping go-cid in that PR there's some (relatively small) refactoring you'll likely need to do in this PR to deal with the relatively small break in go-cid https://github.com/ipfs/go-cid/releases/tag/v0.2.0

Reply:
yeah that's fine I was going to follow up with a PR for that